public/assets/sass/components/_form.scss

```diff
@@ -0,0 +1,60 @@
+.form {
+    display: grid;
+    grid-template-columns: min-content;
+    background-color: $white;
+    justify-content: center;
+
+    @include respond(small-phone) {
+        grid-template-columns: 1fr;        
+    }
+
+    &__signin {
+        background-color: $of-white;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        border-radius: 1rem;
+        padding: 1.3rem;
+        box-shadow: $box-shadow-signin-form ;
```

Comment:
I think this is unique and would always be unique for this `box-shadow`. So, you can consider to inline the color value instead of using a variable.

Reply:
when we add a dark mode , every color will had a light and dark color , 
it will be :
- $box-shadow-signin-form-light
- $box-shadow-signin-form-dark